src/Tasks/Microsoft.NET.Build.Tasks/build/Microsoft.NET.Sdk.targets

```diff
@@ -263,7 +263,12 @@ Copyright (c) .NET Foundation. All rights reserved.
     <ProjectCapability Remove="ReferenceManagerCOM" />
   </ItemGroup>
 
-  <Import Project="$(MSBuildThisFileDirecotry)Microsoft.NET.DisableStandardFrameworkResolution.targets" Condition="'$(DisableStandardFrameworkResolution)' == 'true'" />
+  <!-- Publish capabilities -->
+  <ItemGroup>
+    <ProjectCapability Include="FolderPublish" />
```

Comment:
Can you document this over at https://github.com/Microsoft/VSProjectSystem/blob/master/doc/overview/project_capabilities.md?